src/System.Text.Json/tests/JsonDocumentTests.cs

```diff
@@ -3712,6 +3712,43 @@ private static void AssertContents(string expectedValue, ArrayBufferWriter<byte>
 #endif
                     ));
         }
+
+        [Fact]
+        // Issue #39930
+        public static void Disposing_On_Multiple_Threads_Should_Not_Return_To_ArrayPool_Twice()
```

Comment:
None of the rest of this library using snake case sentences.

```suggestion
        public static void VerifyMultiThreadedDispose()
```

Is a name more in line with this library